resources/views/pages/prompt-tos.blade.php

```diff
@@ -0,0 +1,58 @@
+@extends("templates._page")
+
+@push('head')
+	<style>
+
+		#content {
+			height: 100vh;
+		}
+
+		@media screen and (max-width: 768px) {
+			.btn {
+				width: 100%;
+			}
+		}
+
+		.scroll_box{
+			overflow-y: scroll;
+			height:50%;
```

Comment:
So I see what you're going for here. This should be 50vh, and the `.container` should have its `h-100` removed. In its current state we have an unnecessary scrollbar for the whole page that reveals nothing.